resources/META-INF/plugin_template.xml

```diff
@@ -327,8 +327,8 @@
     <codeInsight.lineMarkerProvider language="Dart" implementationClass="io.flutter.editor.FlutterIconLineMarkerProvider"/>
     <errorHandler implementation="io.flutter.FlutterErrorReportSubmitter"/>
 
-    <toolWindow id="Flutter Outline" anchor="right" icon="FlutterIcons.FlutterDefault"
-                factoryClass="io.flutter.preview.PreviewViewFactory"/>
+<!--    <toolWindow id="Flutter Outline" anchor="right" icon="FlutterIcons.FlutterDefault"-->
+<!--                factoryClass="io.flutter.preview.PreviewViewFactory"/>-->
```

Comment:
how much work is it to commend out the tool window content instead so that a message can be displayed instructing users to open the Structure view?

Reply:
It would be removing the current JComponent which holds the outline and buttons with a message and link to go use the provided Structure view.

We decided to remove the outline, this PR, but I didn't know if we decided on having a few simply point devs to a different window was a net-positive.

Reply:
IMO it would be less jarring for users to have an explanation for why this tool window disappeared than it would be for the tool window to be completely gone. At least for one release cycle would be my preference. @anderdobo what do you think?

Options:
1) completely remove the flutter outline view tool window in IntelliJ / Android Studio
2) remove the outline view content, but leave the tool window. Replace the existing outline content with a message that instructs users to open the IntelliJ structure view to see the tree structure of their code, and to use alt + enter to view a list of available refactorings for a widget.

Reply:
I agree with #2. We should remove the outline view but have messaging in the tool window that guides developers.